lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1552,14 +1552,14 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
         const unsigned RawDataNumWords = BConst->getNumWords();
```

Comment:
GitHub doesn't allow to comment on unchanged code even not so far away from the change one, but there is a comment above on line 1551, which should probably be updated: `arbitrary precision integer` -> `huge arbitrary precision integer`

Reply:
Done